silverback/cluster/client.py

```diff
@@ -189,11 +192,30 @@ def errors(self) -> list[str]:
         handle_error_with_response(response)
         return response.json()
 
-    @property
-    def logs(self) -> list[str]:
-        response = self.cluster.get(f"/bots/{self.id}/logs")
+    def filter_logs(
+        self,
+        log_level: LogLevel = LogLevel.INFO,
+        start_time: datetime | None = None,
+        end_time: datetime | None = None,
+    ) -> list[BotLogEntry]:
+        print("filter_logs:", log_level)
+        query = {"log_level": log_level.name}
+
+        if start_time:
+            query["start_time"] = start_time.isoformat()
+
+        if end_time:
+            query["end_time"] = end_time.isoformat()
+
+        response = self.cluster.get(f"/bots/{self.id}/logs", params=query, timeout=120)
+        print("response:", response)
+        breakpoint()
```

Comment:
Probably still working on it, but just as a reminder
```suggestion
```

Reply:
Oversight, thanks.